### DISCUSSION TITLE
DIA-NN is running 20k+ minutes to create a speclib of rattus novergicus proteome

### DISCUSSION THREAD

**Discussion by ThalisNovo** [vdemichev/DiaNN#1252] [General]
Hi Vadim,
I ran and analysis to myt phd and the SpecLib generation is running for more than 20k minutes and taking 98% of my 64gb RAM without succed. I ran proteomes for fungi with 8k proteins and they got complete, but without the PDF report (I dont know if they are parts of the same issue). 

My computer has:
24 cores (i9 12gen) with 32gb RAM (reaches 64gb due to dual channel);
a 2k SSD NvMe
a geForce 4060Ti 8gb

My data are from a Synapt QToF from Waters in nanoUPLC in DIA in 5 fractions and 3 replicates from control and treated.
I converted the 30 fractions (control and treated in 3 replicates and 5 fractions each) in MzML using the GNPS waters converter and then put them on DIA-NN with the following parameters.

![DIA-NN parameters](https://github.com/user-attachments/assets/8a85cefe-d50c-4dba-9eac-b4041dac2b08)

I've been learning how to use DIA-NN the past two months by myself with the documentation available. Any tips or extra direction would be very helpful. 

Thanks Vadim for the attention

**Comment by ThalisNovo** [comment]
I found out the log for this run in rattus novergicus. I will attach below, it might help to understand the problem. 

[Spectral_Library_Rattus.log.txt](https://github.com/user-attachments/files/17679000/Spectral_Library_Rattus.log.txt)

**Comment by vdemichev** [comment]
Hi Thalis,

The log attached is just for library generation, not the analysis.

To reduce RAM usage, please
- Switch to DIA-NN 1.9.2
- Limit precursor charge range to 2-3
- Try first without variable modifications

Best,
Vadim

**Reply by ThalisNovo** [reply]
Thanks Vadim for the response, Im reanalyzing the data with your suggestions. May I and my supervisor contact you by email? We want to discuss about this analysis and a possible collab :)

**Reply by vdemichev** [reply]
Sure, please email
